pkgs/dmd/build-status.nix

```diff
@@ -116,7 +116,7 @@ let
   };
 in
 mergeVersions [
-  (between "2.092.0" "2.107.2" (version: {
+  (between "2.092.0" latestVersion (version: {
```

Comment:
The old way of manually specifying this led to an annoying troubleshooting detour for me when I didn't realise to increment it ("Why is that Ddoc test we disabled ages ago failing?"), so I thought better to do it this way.